auslib/admin/views/releases.py

```diff
@@ -393,10 +393,28 @@ def get(self, release):
             order_by=[table.timestamp.desc()],
         )
 
+        _revisions = []
+        _mapping = {
+            'data_version': 'data_version',
+            'name': 'name',
+            'product': 'product',
+            'read_only': 'read_only',
+            '_different': '_different',
+            '_time_ago': '_time_ago',
+            'change_id': 'change_id',
+            'changed_by': 'changed_by',
+        }
```

Comment:
It also looks like you're missing "timestamp" from here, which makes the releases history items show up with the wrong time in the UI.